tools/connectivity-check/Cargo.toml

```diff
@@ -14,7 +14,7 @@ prometheus = "0.13.3"
 lazy_static = "1.5.0"
 csv = "1.2.1"
 serde = { version = "1.0", features = ["derive"] }
-
+anyhow = "1.0"
```

Comment:
Hm, I would prefer to handle the errors with the error type of the Rust standard library instead of introducing a dependency on `anyhow`. I don't think using `anyhow` is required here.

See for example https://github.com/0xB10C/peer-observer/blob/90c23c5e67bf04692ca0dc7af4b2bed14150698b/extractor/src/error.rs#L9-L17